src/helix-ui/styles/core/hx-accordion-panel.less

```diff
@@ -0,0 +1,79 @@
+hx-accordion-panel {
+  border-top: 1px solid @gray-300;
+  display: block;
+  padding: 24px 16px;
+
+  header {
+    align-items: center;
+    display: flex;
+    width: 95%;
+  }
+
+  &:hover {
+    .hxStepCounter {
+      background-color: @cyan-500;
+    }
+  }
+
+  &[open] {
+    .hxStepCounter {
+      background-color: @cyan-500;
+      color: @gray-0;
+    }
+  }
+
+  &[disabled] {
+    .hxStepCounter {
+      background-color: @gray-400;
+      color: @gray-700;
+    }
+
+    .hxAccordionPanelTitle,
+    .hxStepTitle {
+      color: @gray-700;
+    }
+
+    &:hover {
+      .hxStepCounter {
+        background-color: @gray-400;
+        color: @gray-700;
+      }
+      header {
+        cursor: inherit;
+      }
+    }
+  }
+}
+
+.hxStepCounter {
```

Comment:
Move the remaining styles to the bottom of `helix-ui.less` (in the "shame" section) until we find a better place to put them.